### PR TITLE
More otie stuff.

### PR DESCRIPTION
**Pull request by Verkister**
-Took out the minimum oxygen requirement from space oties. On test run they all died outside as oxygen was not one of the gases there.
-Moved guard otie to security category in cargo.
-Moved the collared otie from pred crate to its own crate. Bare one is still in the pred crate chances.
-Pet otie got a nifty new box.
-Fixed oties not changing to dead icon when dying while resting.
-Made dino/lizard predmobs able to withstand the outdoors temperature.

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -267,6 +267,7 @@
 #include "code\datums\supplypacks\science.dm"
 #include "code\datums\supplypacks\science_vr.dm"
 #include "code\datums\supplypacks\security.dm"
+#include "code\datums\supplypacks\security_vr.dm"
```
Comment (SpadesNeil) [inline vorestation.dme]: `vorestation.dme:270:error: unable to open "code/datums/supplypacks/security_vr.dm".`

Okay well screw you too, Travis.

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 2]
```diff
@@ -261,6 +261,31 @@
 			return
 			..()
 
+/mob/living/simple_animal/otie/death()
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Wait why are you making an otie death message?

Comment (Verkister) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: To make sure they swap to right icon even when they die while resting.

Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Oooohhh.

Comment (Verkister) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: It only worked if the mob died while standing. Although the oxy tweak here kind of also eliminates the main cause of their passive deaths.

**Pull request merged**